boefjes/boefjes/local.py

```diff
@@ -58,10 +64,14 @@ def run(self, boefje_meta: BoefjeMeta, environment: Dict[str, str]) -> List[Tupl
 class LocalNormalizerJobRunner(NormalizerJobRunner):
     def __init__(self, local_repository: LocalPluginRepository):
         self.local_repository = local_repository
+        self.scheduler_client = SchedulerAPIClient(str(settings.scheduler_api))
 
     def run(self, normalizer_meta, raw) -> NormalizerOutput:
         logger.info("Running local normalizer plugin")
 
+        task_id = str(normalizer_meta.id)
+        self.scheduler_client.patch_task(task_id, TaskStatus.RUNNING)
+
```

Comment:
I don't know if adding this is advisable or desirable. I took note from the docker boefje runner. @Donnype

Reply:
I think this 'self calculation' based on events is less precise than having the boefjesrunner send in its own metrics, but since we might not have those metrics for all runners we need to allow for both paths.

Reply:
Yes, and from the runner perspective, they are still able to update the value, or we can parse it from the received meta data if so desired.